client/app/visualizations/vega/theme.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Vega Theme tweaked for Redash
+ */
```

Comment:
Currently this is just a Copy & Paste of https://github.com/vega/vega-themes/blob/master/src/theme-quartz.ts
still needs more tweaking.